src/pagination/e2e.js

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+/* eslint-env node */
+/* eslint-disable flowtype/require-valid-file-annotation */
+/* global after */
+
+const scenarios = require('./examples-list');
+const {goToUrl} = require('../../e2e/helpers');
+
+const suite = 'Pagination Test Suite';
+
+const selectors = {
+  prevButton: 'button[data-test="prev-button"]',
+  nextButton: 'button[data-test="next-button"]',
+  dropDownButton: 'button[data-test="dropdown-button"]',
+};
+
+describe('The pagination component', () => {
+  after((browser, done) => {
+    browser.end(() => done());
+  });
+
+  it('passes basic accessiblity tests', browser => {
+    goToUrl({
+      suite,
+      test: scenarios.STATEFUL_PAGINATION,
+      browser,
+    })
+      .initAccessibility()
+      .waitForElementVisible('body')
```

Comment:
We should wait for an actual element we want to test instead of the`body` element. In a case component is not rendered for some reason this test will pass successfully.